index.js

```diff
@@ -185,9 +193,53 @@ class Ora {
 			cliCursor.show(this.stream);
 		}
 
+		if (this.consumeStdin && process.stdin.isTTY) {
+			this.stopConsumingStdin();
+		}
+
 		return this;
 	}
 
+	startConsumingStdin() {
+		const me = this;
+		const {stdin} = process;
+
+		this._stdinOldRawMode = stdin.isRaw;
+		this._stdinOldEmit = stdin.emit;
+		this._stdinOldEmitOwnProperty = Object.prototype.hasOwnProperty.call(stdin, 'emit');
+
+		stdin.setRawMode(true);
+		stdin.on('data', noop);
+		stdin.emit = function (event, data) {
+			if (event === 'data') {
+				if (data.includes(0x03)) {
+					if (this.listenerCount('SIGINT') > 0) {
+						this.emit('SIGINT');
+					} else {
+						process.emit('SIGINT');
+					}
```

Comment:
Can you explain the reasoning behind this conditional?

Reply:
Should have done that earlier, forgot the reasoning already.

Then only thing I can find atm is same behavior in `readline`. So, it's either me making a mistake and confusing `readline` implementation with `stdio` implementation or there was an actual reason for that.

Anyhow, removed for now, but probably worthwhile to keep that thing in mind in case we hit some bug regarding that.